src/components/widgets/VocabList.js

```diff
@@ -61,7 +61,6 @@ export default function VocabList() {
           <pre>
             <code className="language-html" ref={codeRef}>
               {String.raw`<dl class="vocab-list">
-  <button>Close All</button>
```

Comment:
Please see line 48 as the <button> here may also be removed. Please test first.

Reply:
I removed this line, but it doesn't appear to be adding the button dynamically. Is there something else I should do to resolve this?
<img width="898" alt="Screenshot 2024-08-30 at 12 59 13 PM" src="https://github.com/user-attachments/assets/324ddb37-ac02-453f-87fa-9f049fc06cb3">